pkg/ui/src/util/api.ts

```diff
@@ -110,7 +110,7 @@ export function toArrayBuffer(encodedRequest: Uint8Array): ArrayBuffer {
 }
 
 // timeoutFetch is a wrapper around fetch that provides timeout and protocol
-// buffer marshalling and unmarshalling.
+// buffer marshaling and unmarshalling.
```

Comment:
this line looks suspicious to me @nvanbenschoten, do we need to explicitly add and error for "unmarshalling"?

Reply:
Good catch! I opened https://github.com/client9/misspell/issues/126 to track this.